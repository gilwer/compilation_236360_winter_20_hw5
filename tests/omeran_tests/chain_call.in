bool d(){
	print("is slavery");
	return true;
}

bool e(){
	print("freedom");
	return d();
}


bool f(){
	print("is peace");
	return e();
}

bool g(){
	print("War");
	return f();
}

void main(){
	bool x = false;
	if (false or x or g()){
		print("--George Orwell");
	}
}
