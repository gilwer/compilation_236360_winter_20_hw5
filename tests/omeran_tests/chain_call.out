War
is peace
freedom
is slavery
--George Orwell
